app/views/layouts/_links.html.erb

```diff
@@ -1,7 +1,7 @@
 <div id="external-links" class="card mb-6" data-controller="card">
   <a href="javascript:void(0)" class="card-header" data-action="card#toggle" aria-label="hide links">
     <span class="card-header-title">
-      Vanjske poveznice s popisima ponude smještaja i prijevoza
+      Kako mogu pomoći?
```

Comment:
Not sure if "Kako mogu pomoći" is the right content but do not have anything better.